docs/home/index.md

```diff
@@ -10,7 +10,89 @@ orphan: true
 }
 </style>
 
-# CrateDB Documentation
+# Welcome to CrateDB
+
+CrateDB is a **distributed SQL database** designed for **real-time analytics
+and search** at scale. Whether you're working with time-series data, full-text
+search, or large volumes of structured and semi-structured data, CrateDB gives
+you the **power of SQL**, the **scalability of NoSQL**, and the **flexibility
+of a modern data platform**.
+
+
+:::::{grid}
+:padding: 0
+
+::::{grid-item}
+:class: rubric-slimmer
+:columns: 6
+
+:::{rubric} Why CrateDB?
+:::
+CrateDB was built for speed, scale, and simplicity:
+
+* **Real-time performance:** Query millions of records per second with sub-second response times.
+* **AI/ML-ready:** Store and serve data for modern AI pipelines.
+* **Search + SQL**: Combine full-text search with rich SQL queries.
+* **Geospatial & time-series**: Native support for IoT, sensor data, and location-based use cases.
+* **Horizontal scalability**: Add nodes effortlessly to handle more data and users.
+* **Resilient and fault-tolerant**: Built-in replication and recovery.
+::::
+
+::::{grid-item}
+:class: rubric-slimmer
+:columns: 6
+
+:::{rubric} What Can You Build?
+:::
+CrateDB is used across industries to power:
+
+* Real-time **dashboards and analytics**
+* Large-scale **IoT platforms**
+* Complex **geospatial applications**
+* Embedded **AI-driven insights**
+* Modern **industrial data backends**
+::::
+
+:::::
+
+
+## Get Started
+
+We've put together some helpful guides to help you set up your CrateDB instance
+quickly and easily. Enjoy reading!
+
+:::{card} {material-outlined}`rocket_launch;1.7em` CrateDB Cloud
+:link: cloud:index
+:link-type: ref
+:link-alt: CrateDB Cloud
+:class-title: sd-fs-5
+
+Start with a fully managed CrateDB instance to accelerate and simplify working
+with analytical data. CrateDB Cloud enables seamless deployment, monitoring,
+backups, and scaling of CrateDB clusters on AWS, Azure or GCPs, eliminating
+the need for direct database management.
+
+With CrateDB Cloud, you can skip infrastructure setup and focus on delivering
+value for your business with a query console, SQL Scheduler, table policies
+and various connectors to import data.
++++
+```{button-link} https://cratedb.com/docs/cloud/tutorials/quick-start.html
+:color: primary
+:expand:
+**Start a forever-free cluster with 8 GB of storage**
+```
+:::
+
+:::{card} {material-outlined}`not_started;1.7em` Getting Started
+:link: getting-started
+:link-type: ref
+:link-alt: Getting started with CrateDB
+:class-title: sd-fs-5
+
+Helpful guides about how to start working with your database cluster.
+:::
+
+## Overview
 
 Welcome to the official CrateDB Documentation. Whether you are a developer,
 database administrator, or just starting your journey with CrateDB, our
```

Comment:
@bmunkholm: Are you referring to the body of this little section re. your comment about redundancies? Here, that is certainly the case, thanks. Let's just remove that body?

Reply:
The fix is part of e16a1344b.